lib/insert-asset-manifest.js

```diff
@@ -0,0 +1,20 @@
+var mergeTrees = require('broccoli-merge-trees');
+var AssetManifestInserter = require('./asset-manifest-inserter');
+
+/**
+ * Given a tree, this function will insert the asset manifest into index.html.
+ *
+ * The `options` object is passed directly into the broccoli plugin.
+ *
+ * @public
+ * @param {Tree} tree
+ * @param {object} options
+ * @return {Tree}
+ */
+module.exports = function insertAssetManifest(tree, options) {
+  options = options || {};
```

Comment:
Yes it is necessary. Your approach assumes that nobody will install this as a node module, `require.resolve()` it, and then directly subclass the Broccoli plugin. The helper and the plugin are distinct possible entry points.